actors/evm/tests/contracts/FilecoinNative.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: Apache-2.0 MIT
+pragma solidity >=0.4.25 <=0.8.17;
+
+contract FilecoinNative {
+    function filecoin_native_method(uint64 method, uint64 codec, bytes calldata params) pure public returns (uint64) {
```

Comment:
cc @ZenGround0 this is how you'd implement the "catchall" for filecoin native methods.

We could also call this something like `filecoin_native_fallback`?

Reply:
👍  I like filecoin_native_fallback or catch_filecoin_native slightly better.

Reply:
why do we have to be so verbose?

Just `invoke_method` should do.

Reply:
Filecoin fallback seems reasonable. 

> why do we have to be so verbose?

We don't want to take any chances of conflicting.

Reply:
erm, its hashed anyway.

Reply:
I know. But things like `invoke_method` are very... generic.

Reply:
Maybe `invoke_filecoin_actor_method`.
Verbose as hell, but less ugly than the proposed names.

Reply:
Final name: handle_filecoin_method.